test/integration/PoolSwap.t.sol

```diff
@@ -1,92 +1,81 @@
+// SPDX-License-Identifier: MIT
 pragma solidity 0.8.25;
 
-import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
-import {Test} from 'forge-std/Test.sol';
-
+import {IERC20} from '@cowprotocol/interfaces/IERC20.sol';
 import {BFactory} from 'contracts/BFactory.sol';
-import {IBPool} from 'interfaces/IBPool.sol';
-
 import {GasSnapshot} from 'forge-gas-snapshot/GasSnapshot.sol';
+import {Test, Vm} from 'forge-std/Test.sol';
+import {IBFactory} from 'interfaces/IBFactory.sol';
+import {IBPool} from 'interfaces/IBPool.sol';
 
 abstract contract PoolSwapIntegrationTest is Test, GasSnapshot {
```

Comment:
I can move this abstract to another file and rename it to `IntegrationBase` or smth, keeping only relevant stuff for both

Reply:
hmmm we could do:
- `SwapBase.t.sol`
- `BPoolSwap.t.sol`
- `BCoWPoolSwap.t.sol`